test/integration/framework/BUILD

```diff
@@ -12,13 +12,16 @@ go_library(
         "master_utils.go",
         "perf_utils.go",
         "serializer.go",
+        "test.server.go",
```

Comment:
weird filename... can we call it `test_server.go`?

Reply:
not sure how that happened